cmd/install.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var installCmd = &cobra.Command{
+	Use:     "install [name] [url]",
+	Short:   "Install a mod from a url this is different from modrinth/curseforge install",
+	Aliases: []string{"add", "get"},
+	Args:    cobra.ArbitraryArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		pack, err := core.LoadPack()
+
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if len(args) != 2 {
+			fmt.Println("Invalid usage: packwiz install [name] [url]")
+			os.Exit(1)
+		}
+		// TODO: consider using colors for these warnings but those can have issues on windows
+		if strings.HasPrefix(args[1], "https://github.com/") {
+			fmt.Println("Consider using packwiz github add", args[1], "instead")
```

Comment:
Could prompt the user if they want to continue? Or just refuse to continue unless the user passes an argument
This should also handle www and non-www / HTTP as well as HTTPS URLs

Reply:
want me to just check if it includes github.com? also www.githubn.com redirects to github.com

Reply:
I meant for all three; a user could manually type it with `www.`. I guess you could parse the URL and get the host from there, would give a nicer error when you pass an invalid URL too

Reply:
Sure I'll do parsing the URL instead